operator/roles/kiali-deploy/tasks/main.yml

```diff
@@ -46,12 +46,6 @@
   when:
   - kiali_vars.istio_namespace == ""
 
```

Comment:
@jmazzitelli the grafana config is changing so there's no explicit "namespace" config anymore and I'm removing this script.

However, now the config has a default url instead: `in_cluster_url: "http://grafana.istio-system:3000"`

I wonder if we should introduce some kind variable replacement mechanism in the config. For instance, we would write something like:

`in_cluster_url: "http://grafana.${ISTIO_NAMESPACE}:3000"`
And that would be automatically replaced in code.